libs/openchallenges/challenge/src/lib/challenge-overview/challenge-overview.component.ts

```diff
@@ -19,8 +19,8 @@ export class ChallengeOverviewComponent {
   organizationCards: OrganizationCard[] = MOCK_ORGANIZATION_CARDS;
   // mockTopics = ['breast', 'cancer'];
 
-  use_default(str: string | null | undefined) {
-    return str === '' || str == null ? 'Not available' : str;
+  useNaIfEmpty(str: string | null | undefined) {
+    return str ?? 'Not available';
```

Comment:
This instruction may return an empty string, which contradicts the name of the function.